libraries/botbuilder-middleware-engine-core/README.md

```diff
@@ -0,0 +1 @@
+# Text Analysis Core Engine
```

Comment:
This one will be published via npm? Won't this cause confusion for people who are looking through `npmjs.com`? Isn't there a way to keep those base classes unpublished and just leverage them in the actual packages. (text analytics and IBM for now).

Reply:
Added the examples folder to `.npmignore` and `CHANGELOG.md` to the repo. I'm publishing the core as a package because at some point in the future, I'd like it be usable outside of this repo too.